src/main/java/org/moltimate/moltimatebackend/alignment/AlignmentUtils.java

```diff
@@ -0,0 +1,77 @@
+package org.moltimate.moltimatebackend.alignment;
+
+import org.biojava.nbio.structure.Group;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+public class AlignmentUtils {
+
+    //ALA, CYS, ASP, GLU, PHE, GLY, HIS, ILE, LYS, LEU, MET, ASN, PRO, GLN, ARG, SER, THR, VAL, TRP, TYR
+
+    public static final HashMap aminoAcidToCharMap = new HashMap<String, String>() {{
```

Comment:
After rebasing with `master`, consider using the new `AminoAcidType` enum to get the Levenshtein characters

Reply:
Done